docs/csharp/reference-semantics-with-value-types.md

```diff
@@ -35,34 +35,34 @@ edit the *csproj* file and add the following node:
 
 You can use either "7.2" or "latest" for the value.
 
-## Specifying `in` parameters
+## Passing arguments by readonly reference
 
 C# 7.2 adds the `in` keyword to complement the existing `ref`
-and `out` keywords when you write a method that passes arguments
-by reference. The `in` keyword specifies that you are passing
+and `out` keywords to pass arguments
+by reference. The `in` keyword specifies passing
 the parameter by reference and the called method does not modify
-the value passed to it. 
+the value. 
 
 This addition provides a full vocabulary to express your design intent. 
-Value types are copied when passed to a called method when you do not
-specify any of the following modifiers. Each of these modifiers specify
+Value types are copied when passed to a called method when you don't
+specify any of the following modifiers. Each of these modifiers specifies
 that a value type is passed by reference, avoiding the copy. Each modifier
 expresses a different intent:
 
 - `out`: This method sets the value of the argument used as this parameter.
 - `ref`: This method may set the value of the argument used as this parameter.
 - `in`: This method does not modify the value of the argument used as this parameter.
 
-When you add the `in` modifier to pass an argument by reference, you declare
-your design intent is to pass arguments by reference to
+Add the `in` modifier to pass an argument by reference and declare
+your design intent to pass arguments by reference to
 avoid unnecessary copying. You do not intend to modify the object used
 as that argument. The following code shows an example of a method
 that calculates the distance between two points in 3D space. 
 
 [!code-csharp[InArgument](../../samples/csharp/reference-semantics/Program.cs#InArgument "Specifying an In argument")]
 
 The arguments are two structures that each contain three doubles. A double is 8 bytes,
-so each argument is 24 bytes. By specifying the `in` modifier, you pass 4-byte
+so each argument is 24 bytes. By specifying the `in` modifier, you pass 4 byte
```

Comment:
Wasn't "4-byte" correct? If not, shouldn't "8-byte" below be fixed too?

Reply:
4-byte is correct.